Tools/DumpRenderTree/cg/PixelDumpSupportCG.h

```diff
@@ -33,17 +33,8 @@
 #include <wtf/RefCounted.h>
 #include <wtf/RetainPtr.h>
 
-#if PLATFORM(WIN)
-#include <windows.h>
-#endif
-
 typedef struct CGContext* CGContextRef;
-
-#if PLATFORM(COCOA)
 typedef void* PlatformBitmapBuffer;
```

Comment:
We don’t need this typedef any more. It existed so we could handle the differences between Cocoa and Windows.

Reply:
Fixed.